src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/SelectArtworkItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { CheckCircleIcon, Box } from "@artsy/palette"
+import EmptyCheckCircleIcon from "@artsy/icons/EmptyCheckCircleIcon"
+
+interface SelectArtworkItemProps {
+  isSelected: boolean
+}
+
+const ICON_SIZE = 24
+
+export const SelectArtworkItem: React.FC<SelectArtworkItemProps> = ({
+  isSelected,
+}) => {
+  return (
+    <Box width={ICON_SIZE} height={ICON_SIZE} ml={10}>
```

Comment:
and it is better to add the left offset at the place where the component is used (this will give us more flexibility if we want to use this component elsewhere. e.g [here](https://github.com/artsy/force/blob/ce8c2824423ec008e98cdffc93e55cfebb702d8b/src/Apps/CollectorProfile/Routes/Saves2/Components/SelectListsForArtworkModal/SelectListItem.tsx#L56-L60))

```javascript
<Box ml="1">
  <SelectArtworkItem isSelected={isSelected} />
</Box>
```